src/Draggable/Sensors/TouchSensor/TouchSensor.js

```diff
@@ -114,7 +114,7 @@ export default class TouchSensor extends Sensor {
    */
   [onTouchHold](event, container) {
     return () => {
-      if (this.touchMoved) {
+      if (this.touchMoved || !event.cancelable) {
```

Comment:
Either the user has moved touch position or the event is cancelable, which means the browser decides to scroll instead.

In both cases we need to bail out before triggering `drag:start`